packages/wallet/frontend/src/lib/api/grants.ts

```diff
@@ -57,7 +87,8 @@ type FinalizeInteractionParams = {
 type FinalizeInteractionResponse = SuccessResponse | ErrorResponse
 
 interface GrantsService {
-  list: (cookies?: string) => Promise<ListGrantsResponse>
+  list: (args?: GrantListArgs, cookies?: string) => Promise<GrantsListResponse>
```

Comment:
Maybe it could be good to rename list -> filter, listAll -> list?

Reply:
Well...it's not really filtering though, it is listing all in a specific pagination setup